concepticondata/conceptlists/Snider-2004-1700.tsv

```diff
@@ -1319,7 +1319,7 @@ Snider-2004-1700-1317	1317	sun	soleil	1343	SUN	37	8	1	6	ENVIRONMENT	NATURE	Sky
 Snider-2004-1700-1318	1318	moon	lune	1313	MOON	37	8	1	6	ENVIRONMENT	NATURE	Sky
 Snider-2004-1700-1319	1319	full moon	pleine lune	3690	FULL MOON	37	8	1	6	ENVIRONMENT	NATURE	Sky
 Snider-2004-1700-1320	1320	new moon	nouvelle lune	3691	NEW MOON	37	8	1	6	ENVIRONMENT	NATURE	Sky
-Snider-2004-1700-1321	1321	eclipse (moon)	éclipse (lunaire)	2285	ECLIPSE	37	8	1	6	ENVIRONMENT	NATURE	Sky
+Snider-2004-1700-1321	1321	eclipse (moon)	éclipse (lunaire)			37	8	1	6	ENVIRONMENT	NATURE	Sky
```

Comment:
ECLIPSE is a more general concept set not restricted to the lunar or solar eclipse.